### DISCUSSION TITLE
HelloWorld.java

### DISCUSSION THREAD

**Discussion by minton55** [graphql-java/graphql-java#3098] [Q&A] [answered]
Considering I cannot find anyone else asking about this, I have to assume that I am missing something.  Whenever I try to compile the HelloWorld.java example (which I cut & pasted from [here](https://graphql.org/code/#java-kotlin), I get an error that "The constructor RuntimeWiring() is undefined" on line 20 and "The method type(String, (<no type> builder) -> {}) is undefined for the type RuntimeWiring" on line 21.

I am assuming that I am doing something stupid here, so if someone could offer a suggestion, I would appreciate it.  Thank you!

**Comment by dondonz** [comment, marked as answer]
Hi @minton55 thanks for reporting that code example is out of date. No, you're not doing anything stupid, that example is very old and that constructor has been replaced with a builder. I'll work out how to update this example on the GraphQL website.

In the meantime, for up to date examples:
* If you want pure GraphQL Java (without Spring), have a look at the [GraphQL Java documentation for the "Hello World" example](https://www.graphql-java.com/documentation/master/getting-started#hello-world) 
* If you want to quickly build a GraphQL service with Spring, see the [Spring for GraphQL getting started tutorial](https://spring.io/guides/gs/graphql-server/), it takes 15 mins to complete and you'll have a working GraphQL service at the end. Spring for GraphQL is the official Spring integration built on top of GraphQL Java

**Reply by dondonz** [reply]
Follow up: I opened a PR to fix the HelloWorld example: https://github.com/graphql/graphql.github.io/pull/1383
